service-statistics-sotka/src/main/java/fi/nls/oskari/control/statistics/plugins/sotka/requests/IndicatorDataJSON.java

```diff
@@ -0,0 +1,39 @@
+package fi.nls.oskari.control.statistics.plugins.sotka.requests;
+
+import java.io.StringWriter;
+
+/**
+ * Request class for SotkaNET statistics query to get indicator data in JSON format.
+ * Data is: [{"indicator" : 4,"region": 231,"year": 2012,"gender": "total","value": 3.4,"absValue": 9}, ... ]
+ * @author SMAKINEN
+ */
+public class IndicatorDataJSON extends SotkaRequest {
+    public final static String NAME = "jsondata";
+
+    public boolean isValid () {
+        return getIndicator() != null && !getIndicator().isEmpty();
+    }
+
+    @Override
+    public String getName() {
+        return NAME;
+    }
+
+    @Override
+    public String getRequestSpecificParams() {
+        StringWriter writer = new StringWriter();
```

Comment:
Could use StringBuilder instead of StringWriter. StringWriter internally uses StringBuffer which is basically synchronized version of StringBuilder. StringBuilder also allows function chaining (e.g. `sb.append("foo).append("bar");`) which is nice.